src/classesToPurge.js

```diff
@@ -0,0 +1,17 @@
+let classes;
+async function getClassesToPurge() {
+  if (classes) return classes;
+  else {
+    classes = await fetch(
+      'https://warp-ds.github.io/css/component-classes/classesToPurge.json',
```

Comment:
Was it considered to put this json file to the Eik CDN? Then we'd make sure that both components.css coming from https://assets.finn.no/pkg/@warp-ds/css/v1/components.css and this file are always in sync, because they would both point to the same version of the css package on Eik. Wdyt?

Reply:
Yes, it would be good. We need to merge the css PR to publish the classes and then use them here. Should not be a problem